examples/angular/package.json

```diff
@@ -11,7 +12,6 @@
     "lint": "ng lint",
     "e2e": "ng e2e"
   },
-  "private": true,
```

Comment:
no longer private?

Reply:
Yep, I moved it up because this `private` was too hard to find 😅

Reply:
🤦‍♀️  sorry, not sure why I missed the green one